README.md

```diff
@@ -217,8 +219,8 @@ jobs:
       auth_project_number: "123456789123"
       service_account: sa-name@project-dev-123.iam.gserviceaccount.com
       image_url: <registry>/<repository>:<tag> or <registry>/<repository>@<digest> # the image created by build job
-      trivy: <optional>
-      tfsec: <optional>
+      trivy: true # default true, optional
+      tfsec: true # default true, optional
       allow_severity_level: medium
```

Comment:
With a security perspective I'd like to argue that a better example would be to tighten the security and not weaken it. Especially for templates and cut and paste examples! :)

Reply:
nvm, i had that completely wrong